src/Altinn.App.Api/Controllers/AuthorizationController.cs

```diff
@@ -123,4 +82,77 @@ public async Task<IActionResult> ValidateSelectedParty(int userId, int partyId)
             return StatusCode(500, $"Something went wrong when trying to validate party {partyId} for user {userId}");
         }
     }
+
+    /// <summary>
+    /// Fetches roles for current party.
+    /// </summary>
+    /// <returns>Boolean indicating if the selected party is valid.</returns>
+    [Authorize]
+    [HttpGet("{org}/{app}/api/authorization/roles")]
+    public async Task<IActionResult> FetchRolesForCurrentParty()
+    {
+        Party? currentParty = await GetCurrentPartyAsync(HttpContext);
+        UserContext userContext = await _userHelper.GetUserContext(HttpContext);
+        int userId = userContext.UserId;
```

Comment:
Could you return this from the helper method instead? So we don't go through `GetUserContext` twice